app/helpers/sidebar_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SidebarHelper
+  def menu_active?(name)
+    controller.controller_name == name ? 'active' : ''
+  end
+
+  def sidebar_sections
```

Comment:
I would move this to a constant, makes more semantic sense to me

SIDEBAR_SECTIONS and then freeze the hash. WDYT?

Reply:
tried, but returns an error:

```undefined local variable or method `boards_path' for SidebarHelper:Module (NameError)```

Also tried including `url_helpers` at the beginning of the module without success:

`include Rails.application.routes.url_helpers`